test/http_content_length_test.dart

```diff
@@ -0,0 +1,240 @@
+// Copyright (c) 2013, the Dart project authors.  Please see the AUTHORS file
+// for details. All rights reserved. Use of this source code is governed by a
+// BSD-style license that can be found in the LICENSE file.
+
+import "dart:io";
```

Comment:
This test should be using the Http implementation from package:http_io/http_io.dart.

Reply:
Oops... this totally defeated the purpose of migrating this test. Done.